src/main/kotlin/nexters/admin/domain/generation_member/SubPosition.kt

```diff
@@ -13,11 +13,16 @@ enum class SubPosition(val value: String?) {
     MANAGER_CMO("CMO"),
     MANAGER_CTO("CTO"),
     MANAGER_CDO("CDO"),
-    NULL(null),
+    NULL(""),
     ;
 
     companion object {
-        fun from(value: String?): SubPosition =
-                values().firstOrNull { it.value == value } ?: throw BadRequestException.wrongSubPosition()
+        fun from(value: String?): SubPosition {
+            if (value == null) {
+                return NULL
+            }
+            return values().firstOrNull { it.value == value }
+                    ?: throw BadRequestException.wrongSubPosition()
+        }
```

Comment:
아예 null을 반환해야되는 상황이라면 아래와 같이 예외를 반환하는 상황을 서비스에 위임하고, 도메인에는 아래와 같이 하는 게 나을 거 같아요
```suggestion
            return values().firstOrNull { it.value == value }
        }
```